### ISSUE TITLE
Unable to locate package fonts-powerline

### ISSUE THREAD

**Issue by feinstein** [powerline/fonts#307]
I am running Ubuntu inside Windows 10. 

When I run: `sudo apt-get install fonts-powerline`

I get this error:

>Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package fonts-powerline

**Comment by jadenPete** [comment]
What is the output of `lsb_release -a`?

**Comment by feinstein** [comment]
>No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic

**Comment by jadenPete** [comment]
Try `sudo apt-get update`. If you still can't install it, post the output of `cat /etc/apt/sources.list`.

**Comment by feinstein** [comment]
well it worked!....the apt-get...I still cant use them in zsh agnoster theme....I am almost certain this is related to the Windows Subsystem for Linux that's running Ubuntu

**Comment by jadenPete** [comment]
I've never used zsh so I'm not sure what the issue is with that, but keep in mind that Windows Subsystem for Linux is run over a Windows console, so special fonts like powerline will probably have to be installed on the Windows side because Ubuntu doesn't render the fonts in that case.

**Comment by feinstein** [comment]
Yeah, I was thinking about this, but I don't know if it will be that simple, because windows will only allow one font for the console, and not several, like a Zsh theme uses.

**Comment by jadenPete** [comment]
It probably works similar to Linux font rendering, in that any font that provides a character can be used. Just because a main font is selected doesn't mean that other fonts won't get factored into the equation. Try installing it in Windows.

**Comment by feinstein** [comment]
I will

**Comment by CSeijiM** [comment]
I used this:

https://medium.com/@slmeng/how-to-install-powerline-fonts-in-windows-b2eedecace58

https://www.hanselman.com/blog/HowToMakeAPrettyPromptInWindowsTerminalWithPowerlineNerdFontsCascadiaCodeWSLAndOhmyposh.aspx
